legion/src/taco_legion_header.cpp

```diff
@@ -757,6 +757,27 @@ void RectCompressedFinalizeYieldPositions::body<3>(Context ctx, Runtime* runtime
   }
 }
 
+template<>
+void RectCompressedFinalizeYieldPositions::body<4>(Context ctx, Runtime* runtime,
```

Comment:
Ah sorry about this. I need to change this code to not require template specializations, as it can be written in a generic way.